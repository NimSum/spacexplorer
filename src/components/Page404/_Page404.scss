section.page-404 {
  position: absolute;
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  height: 100vh;
  z-index: 20;
  width: 100vw;
  background-color: $primary-dark;
  color: $secondary-light;

  h1 {
    font-size: 5rem;
    margin: 0;
  }

  div {
    position: relative;
    width: 70%;
    display: flex;
    flex-direction: column;
    justify-content: center;
    align-items: center;
    margin: 20px;

    img {
      width: 100%;
    }

    h2 {
      position: absolute;
      font-size: 3rem;
    }
  }
}