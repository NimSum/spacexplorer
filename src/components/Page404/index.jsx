import React from 'react'

const Page404 = () =>  (
  <div>
    <h1>Nothing here</h1>
    <h2>:((((((((</h2>
  </div>
)

export default Page404;
