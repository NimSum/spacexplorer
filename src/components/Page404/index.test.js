import React from 'react';
import { shallow } from 'enzyme';
import Page404 from './index';

describe('Page404', () => {
  let wrapper = shallow(< Page404 />)
  
  it('should match component snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  })
})