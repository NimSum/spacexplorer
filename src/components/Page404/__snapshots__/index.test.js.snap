// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page404 should match component snapshot 1`] = `
<section
  className="page-404"
>
  <h1>
    404 Error
  </h1>
  <h2>
    Page Not Found
  </h2>
  <div>
    <img
      alt="exploding rocket"
      src="exploding-rocket.gif"
    />
  </div>
  <Link
    to="/"
  >
    <span>
      Take Me Home
    </span>
  </Link>
</section>
`;
